Jigsaw-app/Jigsaw/Controllers/Home/GameViewController.swift

```diff
@@ -60,6 +60,13 @@ class GameViewController: ORKTaskViewController {
         return step
     }()
     
+    /// Changes continueButtonTitle from "Get Started" to "Chat Now" for the ChatroomInstruction step
+    override func stepViewControllerWillAppear(_ stepViewController: ORKStepViewController) {
+        if self.currentStepViewController?.step?.identifier == "ChatroomInstruction" {
```

Comment:
Consider making the change on the following lines in `RoomProgressViewController` instead of here.

https://github.com/DukeMobileDevCenter/Jigsaw/blob/f3646ad9808da30e3215a5da8250c1f3a1548454/Jigsaw-app/Jigsaw/Controllers/Home/RoomProgressViewController.swift#L407

---

There are some cleanup to do with `RoomProgressViewController` though. I was hoping to separate it into at least 3 files

1. A file that handles all the UI updates on the Room Progress page, where we see all the doors, confetti balls and level information
2. A file to handle the communication with Firebase
3. A file to do the state management for `ORKTaskViewControllerDelegate`, as well as to fix the timing bug on wait page, #116